crates/fj-viewer/src/input/event.rs

```diff
@@ -9,7 +9,17 @@ pub enum Event {
     Key(Key, KeyState),
 
     /// The user scrolled
-    Scroll(f64),
+    Scroll(MouseScrollDelta),
+}
+
+/// Describes a difference in the mouse scroll wheel state.
+pub enum MouseScrollDelta {
+    /// Amount in lines to scroll in the horizontal direction.
+    ///
+    /// Positive values indicate movement forward (away from the user).
+    Line(f64),
+    /// Amount in pixels to scroll in the vertical direction.
+    Pixel(f64),
```

Comment:
A few issues with the documentation here:
- It says `Line` is horizontal, but they're both vertical.
- I believe the sign has the same meaning for both variants, so it probably makes sense to move the comment about positive values to the struct-level doc comment.